pkg/jwt/jwtToken.go

```diff
@@ -0,0 +1,62 @@
+package jwt
+
+import (
+	"encoding/base64"
+	"encoding/pem"
+	"fmt"
+	"time"
+
+	"github.com/golang-jwt/jwt/v5"
+)
+
+type JwtIssuer struct {
+	issuer string
+	secret string
+	ttl    time.Time
+}
+
+func NewJwtIssuer(extras map[string]interface{}) (*JwtIssuer, error) {
```

Comment:
if you have a data type X, then a New function Look like that:
```go
func NewX(field1 string) *X {
  return &X{
    field1: field1,
  }
}
```

and then you can implement a method 
```go
func (j *JwtIssuer) Issue(extras map[string]string) {
  claims := jwt.MapClaims{
    "iss": j.issuer,      
  }
  ...
  }
```

Reply:
yea, I got your point. nice catch ;)